lib/backend/firestore.dart

```diff
@@ -13,3 +19,54 @@ dynamic reference(String documentID) {
           reporter.toFirestore());
   return usersRef;
 }
+
+class ImageStoreMethods {
+  final FirebaseStorage _storage = FirebaseStorage.instance;
+  final FirebaseFirestore _firestore = FirebaseFirestore.instance;
+
+  Future<String> imageToStorage(Uint8List file) async {
+    String id = const Uuid().v1();
+    Reference ref =
+    _storage.ref().child('reports').child(id);
+
+    UploadTask uploadTask = ref.putData(
+        file
+    );
+    TaskSnapshot snapshot = await uploadTask;
+    String imageURL = await snapshot.ref.getDownloadURL();
+    return imageURL;
+  }
+
+  Future<String> uploadPost(
```

Comment:
Convert to void function and use showSnackBar to call snackbars. Make sure rootScaffoldMessengerKey is set in materialapp